app/api/threads/list_threads.feature

```diff
@@ -0,0 +1,172 @@
+      | Charlotte           | EtienneKleinCanWatchAnswer4           | Université           | closed                  | 2021-12-20T00:00:00Z | EtienneKlein is_mine=0 -> List thread ok: can_watch >= answer                                                          |
+      | Charlotte           | EtienneKleinHasValidated6             | Université           | closed                  | 2021-11-00T00:00:00Z | EtienneKlein is_mine=0 -> List thread notok: closed for more than 2 weeks                                              |
+      | Charlotte           | EtienneKleinCanWatchAnswer5           | Université           | closed                  | 2021-11-00T00:00:00Z | EtienneKlein is_mine=0 -> List thread ok: can_watch >= answer                                                          |
+      | Baptiste            | BaptisteCanViewInfo                   |                      |                         |                      | Baptiste is_mine=1 -> notok: can_view < content                                                                        |
+      | Baptiste            | BaptisteCanViewContent1               |                      |                         |                      | Baptiste is_mine=1 -> ok: can_view >= content                                                                          |
+      | Thibaut             | BaptisteCanViewContent2               |                      |                         |                      | Baptiste is_mine=1 -> notok: not the participant                                                                       |
+      | Baptiste            | BaptisteCanViewContentWithDescendants |                      |                         |                      | Baptiste is_mine=1 -> ok: can_view >= content                                                                          |
+      | DavidBowie          |                                       |                      |                         |                      |                                                                                                                        |
+    And the time now is "2022-01-01T00:00:00Z"
+
+  Scenario: Should have all the fields properly set, including first_name and last_name when the access is approved
+    Given I am MarieCurie
+    And there is a group Laboratory referenced by @Laboratory
+    And I am a manager of the group Laboratory
+    And I can watch the group Laboratory
+    And there are the following users:
+      | @reference      | login          | first_name | last_name |
+      | @AlbertEinstein | AlbertEinstein | Albert     | Einstein  |
+      | @PaulDirac      | PaulDirac      | Paul       | Dirac     |
+    And AlbertEinstein the scientist is a member of the group Laboratory
```

Comment:
"the scientist" ? what is this info used for? 🤔

Reply:
fixed, it was to add contextual information, but the scenario was updated since and it doesn't add anything anymore

Reply:
It can improve readability when a scenario features managers and students at the same time, and when the names don't obviously indicate who's the manager and who's the student. We'll see when we have the case